pages/your_tasks/Federated_Learning

```diff
@@ -0,0 +1,47 @@
+---
+title: Federated Learning Best Practices
```

Comment:
Maybe a temporary working title, but I would try to follow our current title structure. What is the task that a researcher/data steward has to tackle, in the context of data management. I think we should also be careful this does not talk to much about software.